studioclass-frontend/src/components/Error.js

```diff
@@ -10,9 +10,10 @@ const styles = {
 }
 
 const Error = ({ error }) => {
+  console.log(error);
   return (
     <Box sx={styles.container}>
-      <Collapse in={error}>
+      <Collapse in={!!error}>
```

Comment:
Consider using `Boolean(error)` here. This is javascript magic and can be confusing for others viewing your code if they're unaware of javascript's type coercion features.